src/Applications/GEOSctm_App/ctm_run.j

```diff
@@ -977,31 +977,37 @@ python bundleParser.py
 setenv YEAR $yearc
 ./linkbcs
 
+# Set OMP_NUM_THREADS
```

Comment:
Again - not a particularly useful comment.  But maybe  "Establish safe default number of OpenMP threads"?